core/txpool/blobpool/blobpool_test.go

```diff
@@ -102,8 +102,9 @@ func (bc *testBlockChain) CurrentBlock() *types.Header {
 	// mainnet ether existence, use that as a cap for the tests.
 	var (
 		blockNumber = new(big.Int).Add(new(big.Int).SetUint64(*bc.config.GetEIP1559Transition()), big.NewInt(1))
-		blockTime   = *bc.config.GetEIP4844TransitionTime() + 1
-		gasLimit    = uint64(30_000_000)
+		// TODO @ziogaschr: consider what to put here using the block number. We can leave it as is, as it's been using in tests only
```

Comment:
@meowsbits @diega what do you think of this one?

Reply:
I think its probably fine to leave until we/anybody else wants to use EIP4844/blobpools in their chain config. The test also assumes `EIP1559` which would potentially need refactoring too.

Reply:
Was thinking to use something like this, but I don't see this code being tested with blockNumbers if we are not going to use this, so let's not add more unused code to maintain.

```
	// If EIP4844 is enabled using block number and it's bigger than EIP1559
	// activation block then use EIP4844's block number
	if eip4844BlockNumber := bc.config.GetEIP4844Transition(); eip4844BlockNumber != nil {
		if blockNumber.Cmp(new(big.Int).SetUint64(*eip4844BlockNumber)) == -1 {
			blockNumber = new(big.Int).SetUint64(*eip4844BlockNumber)
		}
	}
```